StudCity.Frontend/src/pages/verify-email/VerifyEmail.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState }  from 'react';
+import TextField from '../../UI/fields/TextField';
+import { numberValidation } from '../../utils/validators/validators';
+import Svg from '../../components/Svg';
+
+function VerifyEmail() {
+    const [numbers, setNumbers] = useState({
+        number1: "",
+        number2: "",
+        number3: "",
+        number4: "",
+        number5: "",
+        number6: "",
+    });
+
+    const handleChangeNumber = (e, fieldName) => {
+        setNumbers({ ...numbers, [fieldName]: numberValidation(e.target.value) })
+    }
+
+  return (
+    <div className='w-full h-screen flex relative text-center'>
+        <div className='w-6/12 h-screen relative bg-white'>
+            <Svg type='verifyWave' className='rotate-[270deg]  absolute left-[-45%] top-[44%]'/>
```

Comment:
use '-left[45%]' instead of 'left-[-45%]' as docs suggest
PS: in other places as well